sim/core/consumes.go

```diff
@@ -853,6 +853,12 @@ func makeConjuredActivation(conjuredType proto.Conjured, character *Character) (
 					// Restores 900 to 1500 mana. (2 Min Cooldown)
 					manaGain := 900 + (sim.RandomFloat("dark rune") * 600)
 					character.AddMana(sim, manaGain, actionID, true)
+
+					if character.Class == proto.Class_ClassPaladin {
+						// Paladins gain extra mana from self-inflicted damage
+						manaGain = 60 + (sim.RandomFloat("dark rune paladin") * 40)
```

Comment:
Should this just be manaFromDamage := manaGain * 2.0/3.0?

Reply:
This would be simpler and consistent if the self-inflicted damage is consistent with the mana return